docs/pages/getting-started/overview.md

```diff
@@ -1,8 +1,8 @@
-<!-- ---
+
 layout: default
 title: Overview
-permalink: overview
---- -->
```

Comment:
--- -->

Only remove `-->` the other three --- should be present

Reply:
done